codegen/templates/column.go.tpl

```diff
@@ -12,6 +12,9 @@
     {{- if .Options.PrimaryKey}}
       PrimaryKey: true,
     {{- end }}
+    {{- if .Options.IncrementalKey}}
```

Comment:
Do we need this in codegen given we are deprecating this?

Reply:
I think we may as well keep it here until the codegen gets removed (and then this entire file will get removed), don't see any harm in it